src/main/java/com/ably/kafka/connect/config/ChannelSinkConnectorConfig.java

```diff
@@ -206,6 +206,10 @@ public class ChannelSinkConnectorConfig extends AbstractConfig {
 
     public static final String BATCH_EXECUTION_MAX_BUFFER_SIZE_DEFAULT = "1000";
 
+    public static final String BATCH_EXECUTION_MAX_BUFFER_DELAY_MS = "batchExecutionMaxBufferSizeMs";
```

Comment:
Similar to `BATCH_EXECUTION_MAX_BUFFER_SIZE_DOC` below, should there also be a `BATCH_EXECUTION_MAX_BUFFER_DELAY_MS_DOC` string documenting what this is?

Reply:
Yes, and actually it seems I also haven't added this new config to the config builder below. Added now.